x-pack/plugins/maps/public/classes/sources/es_geo_line_source/geo_line_form.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { IndexPattern } from 'src/plugins/data/public';
+import { EuiFormRow } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { SingleFieldSelect } from '../../../components/single_field_select';
+import { getTermsFields } from '../../../index_pattern_util';
+import { indexPatterns } from '../../../../../../../src/plugins/data/public';
+
+interface Props {
+  indexPattern: IndexPattern;
+  onSortFieldChange: (fieldName: string) => void;
+  onSplitFieldChange: (fieldName: string) => void;
+  sortField: string;
+  splitField: string;
+}
+
+export function GeoLineForm(props: Props) {
```

Comment:
would it be possible to validate here so sort and entity-field cannot be the same?

e.g.: we know this would be junk-config
![image](https://user-images.githubusercontent.com/1833023/100952224-1edb3a80-34de-11eb-9dd6-3ec8a3968e5e.png)